navigator_engine/app.py

```diff
@@ -24,6 +25,8 @@ def create_app(config_object=None):
     app.config.from_object(config_object)
     app.url_map.strict_slashes = False
     app.logger.setLevel(app.config.get('LOGGING_LEVEL'))
+    json_logging.init_flask(enable_json=app.config['JSON_LOGGING'])
```

Comment:
I think that json_logging should only by initialized if JSON_LOGGIN set. So:
```
if app.config['JSON_LOGGING']:
  json_logging.init_flask(enable_json=True)
  json_logging.init_request_instrument(app)
```

Reply:
Right!